build-tools-internal/src/main/groovy/org/elasticsearch/gradle/internal/test/AntFixture.groovy

```diff
@@ -10,22 +10,37 @@
 package org.elasticsearch.gradle.internal.test
 
 import org.elasticsearch.gradle.OS
```

Comment:
Though we want to ultimately get rid of AntFixture, I did some tweaks here to get `plugins/discovery-ec2/qa/amazon-ec2` configuration cache compatible as part of this.